package.json

```diff
@@ -157,6 +157,7 @@
     "formik": "^0.11.11",
     "found": "^0.3.14",
     "found-relay": "^0.3.1",
+    "found-scroll": "^0.1.5",
```

Comment:
Ah yeah, this is also the pattern in React Router (of which this `found` is basically a rewrite of v3).